src/components/CardGroup/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ChakraProps, SimpleGrid } from '@chakra-ui/react'
+import { Card } from '@components'
+
+interface CardGroupProps extends ChakraProps {
+  items: SubpageType[]
+  isSimple: boolean
```

Comment:
It's better to make `isSimple` optional and by default `false`. So that you don't have to provide this prop for default component.

> We need to change CardProps as well

Simple:
`<CardGroup items={data} isSimple />`

Default:
`<CardGroup items={data} />`